plugins/outputs/cloudwatch/aggregator_test.go

```diff
@@ -110,13 +110,13 @@ func TestAggregator_MultipleAggregationPeriods(t *testing.T) {
 	aggregator.AddMetric(m)
 
 	assertNoMetricsInChan(t, metricChan)
-	time.Sleep(2 * aggregationInterval)
+	time.Sleep(4 * aggregationInterval)
```

Comment:
I disagree with these changes. We shouldn't increase sleep times in tests just to make the test pass. If it's a flaky test because of timing, then we should figure out what needs to be done in order to make the test more consistent, not sleep longer. This masks the issue, and makes the tests take longer to run.

Reply:
Completely agree, but it would be good to see if increasing the sleep time even makes a difference - perhaps the race condition is elsewhere or it's something else entirely.

@SaxyPandaBear would you mind rerunning the tests with this change in place to see what happens?

Reply:
Sorry for the delay.
I just started the tests.

Reply:
I've pushed a slightly better fix. Still not ideal, but there's only so much time I can devote to this so let's see how it pans out.

Would someone mind kicking off another test run?

Reply:
Making progress. I've pushed a potential ~hack~ fix for another unrelated test

Reply:
Kicked off tests.